src/object.d

```diff
@@ -1102,6 +1104,24 @@ class TypeInfo_Class : TypeInfo
         }
         return o;
     }
+
+   /**
+    * Returns true if the class described by `child` derives from the
```

Comment:
I think the description should explicitly mention that the class is considered a base of itself (making the name a bit misleading).